src/chttpd/src/chttpd_util.erl

```diff
@@ -95,3 +96,20 @@ handle_legacy_config(OriginalHeaders, DefaultHeaderValue) ->
         false ->
             OriginalHeaders
     end.
+
+get_dbs_info_shards(DbName) ->
+    Timeout = fabric_util:request_timeout(),
+    IsolatedFun = fun() -> fabric:get_db_info(DbName) end,
+    try fabric_util:isolate(IsolatedFun, Timeout) of
+        {ok, DbInfo} ->
+            {info, {DbInfo}};
+        {error, Error} ->
+            ErrMsg = "~p : could not get _dbs_info for Db:~p Error:~p",
+            couch_log:error(ErrMsg, [?MODULE, DbName, Error]),
+            {error, not_found}
```

Comment:
I wonder if we should return the actual Error here and then worry about returning `not_found` or serializing the error into json in a different way in the caller after we get the result.